syntax = "proto3";

package chat;

service ChatService {
  rpc ChatStream (stream ChatRequest) returns (stream ChatResponse);
}

message ChatRequest {
  string message = 1;
}

message ChatResponse {
  string reply = 1;
}